contrib/job-scripts/coreudmp-handling/post.d/002-link-core-to-tests

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import errno
+import glob
+import os
+import re
+import sys
+
+if 'AVOCADO_JOB_LOGDIR' not in os.environ:
+    sys.exit(1)
+
+AVOCADO_JOB_LOGDIR = os.environ['AVOCADO_JOB_LOGDIR']
+if not os.path.isdir(AVOCADO_JOB_LOGDIR):
+    sys.exit(2)
+
+COREDUMPS_DIR = os.path.join(AVOCADO_JOB_LOGDIR, 'coredumps')
+if not os.path.isdir(COREDUMPS_DIR):
+    sys.exit(0)
```

Comment:
This is not really necessary. If the COREDUMP_DIR does not exists, the `glob` returns empty set and finishes.